tox.ini

```diff
@@ -75,6 +75,9 @@ setenv =
     AWS_DEFAULT_REGION=us-east-1
     AWS_ACCESS_KEY_ID=accesskey
     AWS_SECRET_ACCESS_KEY=secretkey
+    AZURITE_ACCOUNT_NAME=devstoreaccount1
+    AZURITE_ACCOUNT_KEY=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
```

Comment:
To confirm, this account key value is a fake value, right? If so, would Azurite allow being supplied a shorter string that can be more clearly a dumb value, such as `azuritesecretkey`?

Reply:
Bit puzzled about the unixsockets test suite failing, because seemingly unrelated to this PR, but let me see.

About your comment: Apparently, key has to be base64 encoded. In the added commit, I used a short string that decodes to "azurite".

Reply:
CI failure: Might be the `requests` version.

`master` build on CI used requests-2.31.0, CI build triggered with this PR used requests-2.32.3. 
A (local) run of `tox -e py311-unixsocket` on `master` fails, I forced it to use 2.31.0 version of `requests` and it passed. Can dig deeper, if I have the time. Because `master` fails already, this might not be induced by transitive dependencies of the azure-storage lib introduced here but motivates a separate PR.